ci/documentation/_publish_internal.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env sh
```

Comment:
Script for publishing is almost the same as before.
Git didn't recognize moving with changes.